pytorch_lightning/plugins/training_type/ddp.py

```diff
@@ -380,11 +380,11 @@ def pre_dispatch(self):
         if self.sync_batchnorm:
             self.model = self.configure_sync_batchnorm(self.model)
 
-        self.configure_ddp()
-
         # share ddp pids to all processes
         self._share_information_to_prevent_deadlock()
```

Comment:
could this happen at the very beginning of `pre_dispatch` ? just in case model_to_device or configure sync batchnorm raise errors